src/pages/Jihong/Main.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState, useEffect } from 'react';
+import './Main.scss';
+import ItemList from './MainComponents/ItemList/ItemList';
+import Section from './MainComponents/Section/Section';
+
+const Main = () => {
+  const [items, setItems] = useState([]);
+  const [isClicked, setIsClicked] = useState(items);
+
+  useEffect(() => {
+    fetch('/data/Jihong/ItemListData.json', {
+      method: 'GET',
+    })
+      .then(res => res.json())
+      .then(data => {
+        setItems(data);
+        setIsClicked(data);
+      });
+  }, []);
+
+  const isClick = e => {
+    const filterName = e.target.name;
+    if (filterName === '낮은가격순') {
+      setIsClicked(lowPriceOrder);
+    } else if (filterName === '높은가격순') {
+      setIsClicked(highPriceOrder);
+    } else if (filterName === '최신순') {
+      setIsClicked(items);
+    }
```

Comment:
```suggestion
    if (filterName === '낮은가격순') {
      setIsClicked(lowPriceOrder);
      return;
    } 
    if (filterName === '높은가격순') {
      setIsClicked(highPriceOrder);
      return;
    } 
    if (filterName === '최신순') {
      setIsClicked(items);
      return;
    }
```

else if가 아니라 early return을 사용하면 조금 더 가독성 좋게 코드를 적을 수 있습니다!

Reply:
이부분 잘 해주셨는데 items란 setIsClicked가 다소 겹치는 부분이 있어서 더 개선할 수 있겠네요! 라운지 시간에 찾아와주세요